bedrock/firefox/templates/firefox/nothing-personal/index.html

```diff
@@ -0,0 +1,149 @@
+        </div>
+        {% endcall %}
+      </div>
+
+        {% call browser_border(class='mzp-l-content mzp-t-content-md mzp-t-content-nospace', heading='Nothing Personal') %}
+          <div class="">
+            <h4>You’re great — probably.</h4>
+            <p>We just don’t need a lot of your data to be a <strong>fast, reliable</strong> browser that keeps up with the big guys.</p>
+          </div>
+        {% endcall %}
+
+        <div class="c-thug-life-gif-wrapper mzp-l-content mzp-t-content-md">
+          <div class="c-thug-life-gif c-detached-gif">
+            <img src="{{ static('img/firefox/nothing-personal/thug-life-fox.gif') }}" alt="">
+          </div>
+        </div>
+
+        <div class="sticky-window">
+          <div class="c-thug-life-gif c-attached-gif">
+            <img src="{{ static('img/firefox/nothing-personal/thug-life-fox.gif') }}" alt="">
+          </div>
+
+         {% call browser_border(class='mzp-l-content mzp-t-content-md', heading='SFAQs') %}
+          <div class="">
+            <h4>[Somewhat Frequently Asked Questions]</h4>
+
+            <h5>Are you even any good?</h5>
+            <p>Calm down. And yes. <a href="{{ url('firefox.features.fast') }}">We’re so fast</a> you won’t even notice the difference.</p>
+
+            <h5>What makes you soooo special?</h5>
+            <p>NBD, but people dump their other browsers <a href="{{ url('firefox.features.index') }}">for our features</a> (like <a href="{{ url('firefox.features.picture-in-picture') }}">popping out videos</a> and a <a href="{{ url('firefox.features.pdf-editor') }}">built-in PDF editor</a>).</p>
+
+            <h5>What does any of this have to do with me?</h5>
+            <p>Hey, you can customize Firefox to be all about you with <a href="{{ url('firefox.features.add-ons') }}">add-ons</a> and <a href="{{ url('firefox.features.customize') }}">themes</a>. Happy?</p>
+
+            <h5>Ok, what’s the privacy-catch?</h5>
+            <p><a href="{{ url('firefox.privacy.index') }}"">We don’t keep track of where you go</a> because 1. Yawn, and 2. We really don’t need it.</p>
+          </div>
+          <div class="c-sticky-note c-attached-sticky">
+
+           <span aria-hidden="true">>>></span> {{ download_firefox_thanks(alt_copy='download', button_class='mzp-t-primary mzp-t-lg c-download-firefox-cta', download_location='secondary cta') }} <span aria-hidden="true"> <<<< </span>
+           <br>
+           <span aria-hidden="true">>>></span> {{ download_firefox_thanks(alt_copy='download', button_class='mzp-t-primary mzp-t-lg c-download-firefox-cta', download_location='secondary cta') }} <span aria-hidden="true"> <<<< </span>
+           <br>
+           <span aria-hidden="true">>>></span> {{ download_firefox_thanks(alt_copy='download', button_class='mzp-t-primary mzp-t-lg c-download-firefox-cta', download_location='secondary cta') }} <span aria-hidden="true"> <<<< </span>
+            <p>Ok, bye.</p>
```

Comment:
**issue:** if emulate being on an unsupported OS (e.g. by adding `fx-unsupported` as a class to `<html>`, the the download button messaging breaks out of this small box:

![image](https://github.com/user-attachments/assets/5fa3a93b-2021-4567-9ed6-c8479ae0ffb0)

I'm not sure what the best solution here is but I wonder:

1. Do we really need to repeat the same CTA three times?
2. If we just had it appear once, could we fit the unsupported messaging in the small sticky note?

Alternatively, we could always hide this CTA on unsupported systems?

Reply:
Really good callout!! Thanks for this. I'll just apply `display: none` to the stickynote if it's unsupported :)